src/client/components/componentWithSliders/allSliders/playButton/playButton.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import './playButton.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPlay, faPause } from '@fortawesome/free-solid-svg-icons';
+import PropTypes from "prop-types";
+
+const playIcon = <FontAwesomeIcon icon={faPlay}/>;
+const pauseIcon = <FontAwesomeIcon icon={faPause}/>;
+
+const PlayButton = ({onClick, playing}) => (
+  playing ?
+    <button className="PauseButton" onClick={onClick}>{pauseIcon} Pause</button> :
```

Comment:
here you can create one reusable component button for all you buttons, play, pause and stop, because all of this three buttons are same, only props changing, so i would suggest to create one component for all of them and just pass needed props to it

Reply:
fixed